app/views/custom/layouts/_admin_header.html.erb

```diff
@@ -0,0 +1,42 @@
+<header class="header">
+  <div class="top-links">
+    <div class="expanded row">
+      <%= render "shared/locale_switcher" %>
+
+      <%= link_to root_path, class: "float-right" do %>
+        <%= t("admin.dashboard.index.back", org: setting["org_name"]) %>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="expanded row admin-top-bar">
+    <div class="title-bar" data-responsive-toggle="responsive_menu" data-hide-for="medium">
+      <button class="menu-icon" type="button" data-toggle="responsive_menu"></button>
+      <div class="title-bar-title"><%= t("application.menu") %></div>
+    </div>
+
+    <div class="top-bar">
+      <div class="top-bar-title">
+        <% if namespace == "officing" %>
+          <%= link_to "#", accesskey: "0" do %>
```

Comment:
Why are we adding an accesskey? :thinking:

I'm not sure about the way accesskeys have evolved in the last few years; in the past they were a bit dangerous because they might interfere with keyboard shortcuts :thinking:.

Reply:
This accesskey is inherited from the layouts/_admin_header.html.erb.
It can also be found in the layouts/_header and in our new custom/layouts/_header.html.erb 😅 

I simply tried to maintain the existing logic in the links and focus on displaying the logo correctly.
But I can remove it if we consider it necessary.